apps/prairielearn/python/README.md

```diff
@@ -3,3 +3,5 @@
 This directory contains all the shared python code. Elements can import from this directory to use shared code. You should not import from `test` or from `prairielearn.internal`, as they are not guaranteed to have a stable API.
 
 Questions are executed with their working directory as the question. `zygote.py` (the runner for questions) adds its own working directory to Python's path, so any imports in this directory will be available to questions.
+
+You can view the documentation for the `prairielearn` module [here](https://prairielearn.readthedocs.io/en/latest/python-reference/).
```

Comment:
Links that just say "here" are poor for usability or accessibility. Consider something more like this instead:

```suggestion
View the [documentation for the `prairielearn` module](https://prairielearn.readthedocs.io/en/latest/python-reference/) to learn more about blah blah blah.
```